packages/dos-donts-block/src/components/ImageComponent.tsx

```diff
@@ -70,11 +75,10 @@ const ImageComponent = ({
                 </div>
             </div>
         ) : null;
-    };
 
     return (
         <div className="tw-mb-3">
-            {(!src && isEditing) || isUploadLoading ? (
+            {(!image && isEditing) || isUploadLoading ? (
```

Comment:
Unrelated to your changes but is this really the expected behaviour? That the inject button appears in view mode conditionally when uploading?

Reply:
nice catch, updated it to always require `isEditing`